virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -252,67 +252,76 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-14-known-issues"]
 == Known issues
 
-//CNV-24889 (Avital mentions possible RN in CNV-32472)
-
-//Known issues from 4.13 (and several carried over from 4.12) begin here.Leave in all for now; will be re-reviewed for 4.14.
+[discrete]
+[id="virt-4-14-ki-monitoring"]
+==== Monitoring
+//4.14 Leave in per Stu (fix deferring to 4.15)
+* The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2026733[*BZ#2026733*])
+** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
```

Comment:
Not a now thing as I see this is existing formatting, but using a nested bullet for the workaround seems odd. Curious as to why a list continue (`+`) is not used. Perhaps a global formatting fix after you GA.

Reply:
@ousleyp, okay with you if I change the formatting as suggested?